Tests iOS/MyList/ArchiveFiltersTests.swift

```diff
@@ -87,13 +87,20 @@ class ArchiveFiltersTests: XCTestCase {
 
     func test_archiveView_sortingNoTagFilter_showFilteredItems() {
         app.launch().tabBar.savesButton.wait().tap()
-        let saves = app.saves.wait()
+        let itemCell = app
+            .saves
+            .itemView(matching: "Archived Item 2")
 
-        saves.selectionSwitcher.archiveButton.wait().tap()
+        itemCell
+            .itemActionButton.wait()
+            .tap()
+
+        app.saves.selectionSwitcher.archiveButton.wait().tap()
 
         app.saves.filterButton(for: "Tagged").wait().tap()
         let tagsFilterView = app.saves.tagsFilterView.wait()
 
+        XCTAssertEqual(tagsFilterView.recentTagCells.count, 3)
```

Comment:
Before you do a count, wait for the first item to exist, this is because .count will not wait for cells to exist and can fail instantly.